### DISCUSSION TITLE
Configure some icons to be hidden in the nemo-desktop

### DISCUSSION THREAD

**Discussion by JayBeeDe** [linuxmint/discussions#612] [Questions]
In my enterprise laptop helpdesk is putting some files that are polluting my desktop with useless icons. Is there a way in nemo-desktop settings or dconf to hide some icons ? Removing/Renaming the desktop file doesn't work because puppet overrides it every 5 minutes.

**Comment by 100savage** [comment]
Most likely, if they have puppet setup to override an action every 5 minutes, then anything you do to modify this would be put back with puppet.
